TacTok/models/term_encoder.py

```diff
@@ -145,6 +165,43 @@ def get_vocab_idx(self, node, localnodes):
         else:
             return vocab.index('<unk-ident>')
 
+    def normalize_length(self, length, pad, seq):
+        if len(seq) > length:
+            return seq[:length]
+        elif len(seq) < length:
+            return seq + [pad] * (length - len(seq))
+        else:
+            return seq
+
+    def encode_identifiers(self, nodes):
+        encoder_initial_hidden = torch.zeros(1, len(nodes), 
+                                             self.opts.ident_vec_size)
+        node_identifier_chunks = \
+          torch.tensor([self.normalize_length(
+                          self.opts.max_ident_chunks,
+                          0,
+                          # Add 1 here to account for the padding value of zero
+                          [tok + 1 for tok in 
+                           self.name_tokenizer.tokenize_to_idx(
+                             node.children[0].data)]
+                           if (node.data == "constructor_var" or
```

Comment:
I'd also comment here. We will want to use the latest static methods for these once we merge in the other changes Timothy and I have made recently. These strings are really brittle/sensitive to changes so we don't want to refer to them in text too much. Since you're merging first, though, it'd be good to just comment explaining what those strings are and where you got them from, so that when we do the merge we know how to make the appropriate changes.